### ISSUE TITLE
docstring for inv(::Puiseux)

### ISSUE THREAD

**Issue by kalmarek** [Nemocas/AbstractAlgebra.jl#213]
points to inverse of Matrix:

https://nemocas.github.io/AbstractAlgebra.jl/puiseux.html#Division-1

**Comment by thofma** [comment]
Maybe fixed by https://github.com/Nemocas/AbstractAlgebra.jl/pull/202?

**Comment by wbhart** [comment]
I am not sure about that. But let's check, when it's finally merged.

Bill.

**Comment by thofma** [comment]
But we should not mess with the docs while @sachinkm308 is working on them.

**Comment by wbhart** [comment]
Agreed. By the way, he is done with AbstractAlgebra. I have to review all his changes again as soon as I find the time.
